src/lib/RepStatus.js

```diff
@@ -106,26 +106,31 @@ class RepStatus extends EventEmitter {
      */
     handleResponse() {
         const status = RepStatus.getStatus(this.representation);
-        let errorCode;
+        const errCode = RepStatus.getErrorCode(this.representation);
+        let errorMessage;
+        let error;
```

Comment:
This section changed to allow readable error codes with translated display messages